src/tests/test_utils.ts

```diff
@@ -9,116 +9,99 @@ import {
 } from "../types.ts";
 
 export const FAKE_ID = "ABC123";
-export const generatePayload = (
-  id: string,
+export const generateFunctionExecutedPayload = (
+  callback_id: string,
+  enterprise_id?: string,
```

Comment:
Our tests currently don't use this new `enterprise_id` argument, but I think maybe we should add some tests that specify this argument, then these fake payload creation helpers would simulator execution in an enterprise grid environment.